lib/api/v3/work_packages/work_package_payload_representer.rb

```diff
@@ -35,6 +35,25 @@ class WorkPackagePayloadRepresenter < WorkPackageRepresenter
 
         cached_representer disabled: true
 
+        property :file_links,
+                 exec_context: :decorator,
+                 getter: ->(*) {},
+                 setter: ->(fragment:, **) do
+                   next unless fragment.is_a?(Array)
+
+                   ids = fragment.map do |link|
+                     ::API::Utilities::ResourceLinkParser.parse_id link['href'],
+                                                                   property: :file_link,
+                                                                   expected_version: '3',
```

Comment:
what is that `version` information?

Reply:
That is the api version information. We have v3 in our case.